src/controller/index.ts

```diff
@@ -145,7 +145,13 @@ export default class SimulariumController {
                 this.onError
             );
             this.remoteWebsocketClient = webSocketClient;
-            this.simulator = new RemoteSimulator(webSocketClient, this.onError);
+            this.simulator = new RemoteSimulator(
+                webSocketClient,
+                netConnectionConfig.useOctopus
```

Comment:
I think what you are trying to do here is `!!netConnectionConfig.useOctopus` -- does that seem right?

Reply:
- [ ] change to !!netConnectionConfig.useOctopus